gaozhewei/server/src/ser_server.cpp

```diff
@@ -0,0 +1,27 @@
+#include "ros/ros.h"
+#include "server/name.h"
+#include "server/tim.h"
+
+void timeInfoCallback(const ser::tim::ConstPtr& msg) {
+    ROS_INFO("%s %d", msg->myName.c_str(), msg->tim);
+}
+
+bool serCallback(const ser::name::Request &req,
+                 const ser::name::Response &res) {
+    ROS_INFO("Topic:%s", req.topic_name);
+    res.result = "OK";
+    return true;
+}
+
+int main(int argc, char **argv) {
+    char *i;
+    ros::init(argc, argv, "my_time_server");
+    ros::NodeHandle node;
+    ros::ServiceServer name_service =
+            node.advertiseService("my_time_server", serCallback);
+    ros::Subscriber time_info_suc =
+            node.subscribe("time_info", 10, timeInfoCallback);
+    ROS_INFO("Ready!");
+    ros::spin();
+    return 0;
+}
```

Comment:
**[cpplint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br> Could not find a newline character at the end of the file.  [whitespace/ending_newline] [5]